app/src/main/java/com/sopt/now/presentation/login/LoginViewModel.kt

```diff
@@ -4,36 +4,40 @@ import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import com.sopt.now.MyApplication
+import com.sopt.now.R
 import com.sopt.now.presentation.model.User
 
 class LoginViewModel : ViewModel() {
-    private val _loginState: MutableLiveData<Boolean> = MutableLiveData(false)
-    val loginState: LiveData<Boolean> = _loginState
+    private val _loginState: MutableLiveData<LoginState> = MutableLiveData(LoginState.Empty)
+    val loginState: LiveData<LoginState> = _loginState
```

Comment:
```suggestion
    val loginState: LiveData<LoginState> get() = _loginState
```
이런 식으로 쓸 수도 있답니당! 두 차이점에 대해 찾아보면 좋을 것 같네용